src/__tests__/__snapshots__/footer.test.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Need to think about how to simplify the dev process... @guzman-raphael To check.